pkg/ocibundle/native/bundle_linux.go

```diff
@@ -81,6 +89,15 @@ func OptImgCache(ic *cache.Handle) Option {
 	}
 }
 
+// OptTempDir sets the parent temporary directory for temporary files generated
+// outside of the assembled bundle.
+func OptTmpDir(tmpDir string) Option {
```

Comment:
Right - at present it will always be "" so it'll do `os.MkdirTemp("", "xxxxx")` and use the system tempdir / env var.

It was added to expose it, so it can be wired up properly to the `SINGULARITY_TMPDIR` env var etc. later. I should open an issue to remind me.